src/components/Synthetics/TVChart/useChartHeaderFormattedValues.tsx

```diff
@@ -2,52 +2,121 @@ import { useMemo } from "react";
 
+import { AvailableLiquidityTooltip } from "./components/AvailableLiquidityTooltip";
+import { NetRate1hTooltip } from "./components/NetRate1hTooltip";
+
 export function useChartHeaderFormattedValues() {
   const dailyVolumeValue = use24hVolume();
   const info = useSelector(selectChartHeaderInfo);
 
-  const longOIValue = useMemo(() => {
+  const [longOIValue, longOIPercentage] = useMemo(() => {
     if (info?.longOpenInterestPercentage !== undefined && info.openInterestLong !== undefined) {
-      return (
-        <span className="whitespace-nowrap">
-          {formatAmountHuman(info?.openInterestLong, USD_DECIMALS)} (
-          {formatPercentageDisplay(info.longOpenInterestPercentage)})
-        </span>
-      );
+      return [
+        <span key="long-oi-value" className="whitespace-nowrap">
+          ${formatAmountHuman(info?.openInterestLong, USD_DECIMALS)}
+        </span>,
+        formatPercentageDisplay(info.longOpenInterestPercentage),
+      ];
     }
 
-    return "-";
+    return ["...", null];
   }, [info?.longOpenInterestPercentage, info?.openInterestLong]);
 
-  const shortOIValue = useMemo(() => {
+  const [shortOIValue, shortOIPercentage] = useMemo(() => {
     if (info?.longOpenInterestPercentage !== undefined && info.openInterestShort !== undefined) {
-      return (
-        <span className="whitespace-nowrap">
-          {formatAmountHuman(info?.openInterestShort, USD_DECIMALS)} (
-          {formatPercentageDisplay(100 - info.longOpenInterestPercentage)})
-        </span>
-      );
+      return [
+        <span key="short-oi-value" className="whitespace-nowrap">
+          ${formatAmountHuman(info?.openInterestShort, USD_DECIMALS)}
+        </span>,
+        formatPercentageDisplay(100 - info.longOpenInterestPercentage),
+      ];
     }
 
-    return "-";
+    return ["...", null];
   }, [info?.longOpenInterestPercentage, info?.openInterestShort]);
 
-  const liquidityLong = info?.liquidityLong ? formatAmountHuman(info?.liquidityLong, USD_DECIMALS) : "-";
-  const liquidityShort = info?.liquidityShort ? formatAmountHuman(info?.liquidityShort, USD_DECIMALS) : "-";
+  const liquidityLong = useMemo(() => {
+    const liquidity = info?.liquidityLong;
+
+    if (liquidity === undefined) {
+      return "...";
+    }
+
+    return (
+      <TooltipWithPortal
+        disableHandleStyle
+        handle={`$${formatAmountHuman(liquidity, USD_DECIMALS)}`}
+        position="bottom-end"
+        renderContent={() => <AvailableLiquidityTooltip isLong />}
+      />
+    );
+  }, [info?.liquidityLong]);
+
+  const liquidityShort = useMemo(() => {
+    const liquidity = info?.liquidityShort;
+
+    if (liquidity === undefined) {
+      return "...";
+    }
+
+    return (
+      <TooltipWithPortal
+        disableHandleStyle
+        handle={`$${formatAmountHuman(liquidity, USD_DECIMALS)}`}
+        position="bottom-end"
+        renderContent={() => <AvailableLiquidityTooltip isLong={false} />}
```

Comment:
JFYI, we also have content property
```suggestion
        content={<AvailableLiquidityTooltip isLong={false} />}
```